### ISSUE TITLE
is possible download the automatic build of travis?

### ISSUE THREAD

**Issue by pietrondo** [kr15h/ofxPiMapper#144]
Hi!
Is possible download the automatic build of travis-ci?

thx

**Comment by kr15h** [comment]
Yes. You can find it under releases. And it is linked to the website as well. I am currently working on improvements of the Travis CI script. Since a few days ago openFrameworks v0.10.0 was released, etc. ect. tce...

**Comment by pietrondo** [comment]
could be userfull when i need to download the windows client..
